STUDIO/org.openl.rules.repository.azure/src/org/openl/rules/repository/azure/FileInfo.java

```diff
@@ -0,0 +1,22 @@
+package org.openl.rules.repository.azure;
+
+public class FileInfo {

```

Comment:
package-private access?

Reply:
It's required to be public to be serializable by yaml